MOAS-LMS/Views/Course/_Activity.cshtml

```diff
@@ -32,9 +32,15 @@
                     @Html.ActionLink("Edit", "Edit", "Activity", new { id = Model.Id }, null) |
                     @Html.ActionLink("Delete", "Delete", "Activity", new { id = Model.Id }, null)
                 </span>
+                <p></p>
                 <form action="/Document/UploadToActivity/@Model.Id" method="post" enctype="multipart/form-data">
+                    @*<input type="file" name="file" value="" />
+                    <input type="submit" value="Upload" />*@
                     <input type="file" name="file" value="" />
-                    <input type="submit" value="Upload" />
+                    <div class="fileUpload btn btn-primary" >
+                        <span>Upload</span>
+                        <input type="submit" class="upload" />
+                    </div>
```

Comment:
Du glömde upload-knappen för att lämna in assignments, precis under de här raderna.

Det skulle kanske också se bättre ut om du gav `<input>` `display: inline-block`?

Reply:
jag försökte få "upload"-knappen att ligga bredvid "choose file" tidigare, men det krockade iaf med texten av den 'valda' filen, så jag valde att flytta "upload"-knappen med css.
![choose_file](https://user-images.githubusercontent.com/38253015/43709921-75c6a8ce-996e-11e8-99b4-29457448d9d9.png)